department-of-veterans-affairs/caseflow
# Show task canceler in case timeline

Resolves CM ticket AC
<img width="649" alt="Screen Shot 2020-05-15 at 2 36 55 PM" src="https://user-images.githubusercontent.com/45575454/82093113-7f349700-96c8-11ea-9f32-49111964bb73.png">


### Description
Shows who cancelled a task in case timeline. Works for any task that is cancelled though task actions.

### Acceptance Criteria
- [x]  Task canceler is shown in the case timeline entry for the cancelled task

### Testing Plan
1. Sign in as BVALSPORER
1. Go to your queue
1. Click on any case
1. Cancel your task
1. Go back to case details and look at the case timeline
1. Confirm your css id shows as the canceler.

### User Facing Changes
BEFORE|AFTER
 ---|---
<img width="499" alt="Screen Shot 2020-05-15 at 4 22 48 PM" src="https://user-images.githubusercontent.com/45575454/82093082-6deb8a80-96c8-11ea-9669-5abbe6188fca.png">|<img width="493" alt="Screen Shot 2020-05-15 at 4 23 14 PM" src="https://user-images.githubusercontent.com/45575454/82093087-7217a800-96c8-11ea-9bbb-492adfab7320.png">
